me.hyliu.fluentreader.yml

```diff
@@ -0,0 +1,90 @@
+branch: stable
+sdk: org.freedesktop.Sdk
+base: org.electronjs.Electron2.BaseApp
+base-version: "20.08"
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.node14
+command: start-fluent-reader
+separate-locales: false
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --device=dri
+  - --socket=pulseaudio
+  - --share=network
+modules:
+  # First step is to install Node to /app/node, that way it can be accessible outside of the sdk
+  # environment. install-sdk.sh is used because install.sh omits npm.
+  # (This does not need to be done for electron-builder, see
+  # electron-webpack-quick-start's build.electron.webpack.ElectronWebpackQuickStart.yaml for an
+  # explanation.)
+  # - name: node
+  #   buildsystem: simple
+  #   build-commands:
+  #     - '/usr/lib/sdk/node14/install-sdk.sh'
+
+  # Now is the quickstart module.
+  - name: fluent-reader
+    buildsystem: simple
+    build-options:
+      # Add the node bin directory.
+      append-path: /usr/lib/sdk/node14/bin
+      env:
+        # Set the Electron cache directory.
+        # (The directory format is: /run/build/MODULE_NAME/flatpak-node/electron-cache)
+        # ELECTRON_CACHE: '/run/build/fluent-reader/flatpak-node/electron-cache'
+        XDG_CACHE_HOME: /run/build/fluent-reader/flatpak-node/cache
+        # Sets the directory where Node is located so way npm won't download the headers.
+        # npm_config_nodedir: '/usr/lib/sdk/node14'
+    build-commands:
+      # Have Yarn use the offline mirror.
+      - "npm install --offline --prefix=fluent-reader --cache=$FLATPAK_BUILDER_BUILDDIR/flatpak-node/npm-cache"
+      # Download the packages.
+      # If you were using npm with electron-webpack/electron-builder, then the above two commands
+      # would look more like the npm commands in the vanilla-quick-start manifest, just without
+      # the --prefix.
+
+      # Run electron-builder, passing the architecture arguments to it.
+      # Note that the -- is important; without that, the argument will be passed to
+      # yarn instead of electron-builder.
+      - ". flatpak-node/electron-builder-arch-args.sh; npm run --prefix=fluent-reader --offline build; npm run --prefix=fluent-reader --offline package-linux-unpacked -- $ELECTRON_BUILDER_ARCH_ARGS"
+      # Copy the resulting, unpacked directory to /app.
+      # (A glob is used because the directory name may contain the current arch.)
+      - "cp -r fluent-reader/bin/linux/*/linux*unpacked /app/fluent-reader"
+      # If you passed --electron-non-patented-ffmpeg, you could install it like this:
+      # - 'install -Dm 755 flatpak-node/libffmpeg.so -t /app/electron-webpack-quick-start'
+      # Install the wrapper script to start it.
+      - "install -Dm 755 start-fluent-reader.sh /app/bin/start-fluent-reader"
+      - install -Dm644 fluent-reader/dist/icons/logo.svg /app/share/icons/hicolor/scalable/apps/me.hyliu.fluentreader.svg
+    sources:
+      - type: git
+        url: https://github.com/xcffl/fluent-reader/
```

Comment:
Please don't use downstream fork. Add patches instead if needed.

Reply:
Oh... that's by mistake :laughing: . Thanks for notice. Was used for local testing. But before https://github.com/yang991178/fluent-reader/pull/133 gets merged, it won't build.

Corrected.

Reply:
You can use `git format-patch -1` or something to create patch with your changes. Or just [download](https://github.com/yang991178/fluent-reader/pull/133.patch) your PR as a patch.